src/test/java/org/swisspush/redisques/RedisQuesProcessorDelayedExecutionTest.java

```diff
@@ -91,12 +91,10 @@ public void queueProcessorShouldBeNotifiedWithNonLockedQueue(TestContext context
             processorCalled.set(true);
         });
 
-        eventBusSend(buildEnqueueOperation(queue, "hello"), reply -> {
-            context.assertEquals(OK, reply.result().body().getString(STATUS));
-        });
+        eventBusSend(buildEnqueueOperation(queue, "hello"), reply -> context.assertEquals(OK, reply.result().body().getString(STATUS)));
```

Comment:
I see no benefit decreasing readability here.

Reply:
Yep, github-first reviewers like me have to scroll right. Keep in the 120 chars, move the result to the line below.

Reply:
I will move the result to a new line keeping the 120 chars limit but also keep _expression lambda instead_ of _statement lambda_